Classes/Neos/Neos/Ui/Domain/Model/Feedback/Operations/RemoveNode.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Neos\Neos\Ui\Domain\Model\Feedback\Operations;
+
+use Neos\Neos\Ui\Domain\Model\FeedbackInterface;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+use Neos\Flow\Mvc\Controller\ControllerContext;
+
+class RemoveNode implements FeedbackInterface
```

Comment:
Actually, `DocumentNodeCreated` is currently the exception among the feedback classes. We have stuff there like `UpdateWorkspaceInfo` or `RenderContentOutOfBand` and conceptually, the idea was, that those "feedbacks" are commands rather than events, so actually `DocumentNodeCreated` should be renamed to `AddDocumentNode`.

The idea was, that the backend should indeed have a mechanism, to tell the UI what to do (in a sense). The backend merely tells the UI about side effects, that happened during the operation.

But I get, that this might be confusing and we should discuss, whether this is still a good idea.